.github/workflows/maven-publish.yml

```
@@ -0,0 +1,31 @@
+name: Maven Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Java for publishing to Maven Central
+      uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        server-id: ossrh
+        server-username: MAVEN_USERNAME
+        server-password: MAVEN_PASSWORD
+        gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+        gpg-passphrase: MAVEN_GPG_PASSPHRASE
+
+    - name: Build and publish to Maven Central
+      run: mvn -B clean deploy -P release
```

Jeg foretrekker at man bruker den lange versjonen av argumenter i slike filer siden dette gjør det enklere å vite hva som er hva uten å måtte ha veldig dyp kjennskap til programmene.

```suggestion
      run: mvn --batch-mode --errors clean deploy -P release
```